packages/web/src/services/AudiusBackend.ts

```diff
@@ -109,9 +141,23 @@ export const AuthHeaders = Object.freeze({
   Signature: 'Encoded-Data-Signature'
 })
 
+type SnakeToCamel<S extends string> = S extends `${infer T}_${infer U}`
+  ? `${T}${Capitalize<SnakeToCamel<U>>}`
+  : S
+
+type SnakeKeysToCamel<T> = {
+  [K in keyof T as SnakeToCamel<Extract<K, string>>]: T[K]
+}
+
+type DiscoveryEndpoint = (...args: any) => { queryParams: Record<string, any> }
```

Comment:
no worries but you were wanting to address anys?

Reply:
Yes, this one could just be a couple type params. But I'm gonna leave it for now because it's really only used for `DiscoveryApiParams` helper type

The other `any`s can be fixed once the libs typing stuff is merged!